src/joint.dia.element.js

```diff
@@ -694,8 +694,6 @@ joint.dia.ElementView = joint.dia.CellView.extend({
 
     pointerdown: function(evt, x, y) {
 
-        this.model.trigger('batch:start');
-
         // target is a valid magnet start linking
         if (evt.target.getAttribute('magnet') && this.paper.options.validateMagnet.call(this.paper, this, evt.target)) {
 
```

Comment:
I do see your point about the 'add' happening before the batch, however.. I will add it back in